### PR TITLE
Add launcher csv

### PR DESCRIPTION
**Pull request by JameelB**
## What
Adds a Launcher CSV in preparation for adding Launcher to the Integreatly operator, and updates RHSSO version

- [x] Add Launcher CSV
     - This is generated from the [launcher-operator](https://github.com/fabric8-launcher/launcher-operator)
- [x] Add package
- [x] Update the makefile to include `push/launcher` and add it to `push/all`

## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: Launcher (0.1.2), RHSSO(1.8.2) and Integreatly Operator (1.6.0) should be available in your app registry
2. Create an `OperatorSource` with your app registry
 - **Verify**: Launcher and RHSSO should become available in the Operator Hub
 - **Verify**: Launcher and RHSSO CR should be available in the developer catalog
3. Install Launcher to a specific namespace
 - **Verify**: An Install plan should be created
 - **Verify**: Launcher and RHSSO operator should be deployed successfully

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,6 +20,9 @@ push/3scale:
 	operator-courier verify 3scale
 	-operator-courier push 3scale/ $(REPO) 3scale $(3SCALE_VERSION) "$(AUTH_TOKEN)"
 
+push/launcher:
+	operator-courier verify launcher
+	-operator-courier push launcher/ $(REPO) launcher $(LAUNCHER_VERSION) "$(AUTH_TOKEN)"
```
Comment (philbrookes) [inline Makefile]: this line has changed format recently, can you copy it from the new format on master

**Comment by philbrookes** [comment]
it's worth waiting until you have everything working in your own app registry and image registry before merging this, so you can update the integreatly manifest at the same time.

**Comment by JameelB** [comment]
@philbrookes Sounds good. Marking this as WIP until changes for the integreatly operator has been added

File: launcher/0.0.2/launcher-operator.0.0.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,180 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (philbrookes) [inline launcher/0.0.2/launcher-operator.0.0.2.clusterserviceversion.yaml]: we need to include the product name in the version-named directory.

i.e. rename directory `0.0.2` to `launcher-0.0.2`

File: Makefile [hunk 3]
```diff
@@ -6,11 +6,11 @@ RHSSO_VERSION=1.7.0
 CODEREADY_VERSION=1.2.2
 FUSE_VERSION=1.7.0
 NEXUS_VERSION=0.8.1
-TUTORIAL_WEB_APP_VERSION=0.0.21
```
Comment (dimitraz) [inline Makefile]: I think this was accidentally deleted

**Comment by dimitraz** [comment]
@austincunningham can you bump the integreatly operator version as well?

File: Makefile (hunk 3 above)
Comment (austincunningham) [reply Makefile]: :+1:

**Comment by dimitraz** [comment]
@austincunningham needs a rebase

File: Makefile [hunk 4]
```diff
@@ -1,16 +1,17 @@
 AMQONLINE_VERSION=0.28.0
-INTEGREATLY_VERSION=1.5.10
+INTEGREATLY_VERSION=1.6.0
 3SCALE_VERSION=0.3.0
 AMQSTREAMS_VERSION=1.1.0
 RHSSO_VERSION=1.8.1
```
Comment (austincunningham) [inline Makefile]: Missed the version bump here

**Pull request merged**